document_page_approval/tests/test_document_page_approval.py

```diff
@@ -46,7 +44,7 @@ def test_change_request_approve(self):
 
         # It should automatically be in 'to approve' state
         self.assertEqual(chreq.state, "to approve")
-        self.assertNotEqual(chreq.content, page.content)
+        self.assertEqual(chreq.content, page.content)
```

Comment:
Why are you changing the test?

Reply:
Excuse me, I didn't remember that I had done that. In Odoo 13.0 the calculated fields have been improved and now we have to force the calculation. New rebase in progress

Reply:
Isn't `flush` enough? I mean, do `chreq.flush()` before.
`self.env.add_to_compute(page._fields["content"], page)` shouldn't be necessary

Reply:
Flush is not enough. I have tried previously but it didn't work